Makefile

```diff
@@ -29,8 +29,8 @@ docker-push:
 
 test-docker:
 	@# run tests inside a docker container
-	docker run --rm -v "$(PWD)":/host -w /host --network="host" finestructure/spi-base:0.3.1 \
-	  bash -c "apt-get update && apt-get install -y unzip && make test"
```

Comment:
Could you give `make test-docker` on your machine another spin please to confirm it's all working locally as well?

Reply:
I get a rather unusual error:

```
> make test-docker
docker run --rm -v "/Users/dave/Developer/SwiftPackageIndex/Server":/host -w /host --network="host" finestructure/spi-base:0.4.0 \
	  make test
swift test --enable-test-discovery --enable-code-coverage
warning: '--enable-test-discovery' option is deprecated; tests are automatically discovered on all platforms
[1/1] Planning build
[1/1] Planning build

* Build Completed!
[1/1076] Compiling Logging LogHandler.swift
[2/1076] Compiling Logging Locks.swift
[3/1076] Compiling Logging Logging.swift
[4/1076] Compiling Plot ComponentBuilder.swift
[5/1076] Compiling Plot ComponentContainer.swift
[6/1076] Compiling Plot HTMLComponents.swift
[7/1076] Compiling Plot HTMLElements.swift
[8/1076] Compiling Plot HTMLFormContentType.swift
[9/1076] Compiling Plot HTMLFormMethod.swift
ERRO[0022] error waiting for container: invalid character 'u' looking for beginning of value 
make: *** [test-docker] Error 125
```

Reply:
Although, something may be up 😬

I run it again and get this:

```
> make test-docker
docker run --rm -v "/Users/dave/Developer/SwiftPackageIndex/Server":/host -w /host --network="host" finestructure/spi-base:0.4.0 \
	  make test
docker: Error response from daemon: Bad response from Docker engine.
See 'docker run --help'.
make: *** [test-docker] Error 125
```

Also, it seems to have deleted all my Docker images 🚀

Reply:
Even after a reboot:

```
> make test-docker
docker run --rm -v "/Users/dave/Developer/SwiftPackageIndex/Server":/host -w /host --network="host" finestructure/spi-base:0.4.0 \
	  make test
docker: Error response from daemon: dial unix docker.raw.sock: connect: connection refused.
See 'docker run --help'.
make: *** [test-docker] Error 125
```

Something is horribly wrong with my Docker. I'll reinstall it.